### PR TITLE
[TASK] Refactor QueryView->tableWrap to <pre> (#578)

### LINKED ISSUE #578: Deprecation: #77557 - Method QueryView->tableWrap()
Deprecation: #77557 - Method QueryView->tableWrap()

https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/8.3/Deprecation-77557-MethodQueryView-tableWrap.html

.. include:: ../../Includes.txt

===================================================

Deprecation: #77557 - Method QueryView->tableWrap()

===================================================

See :issue:`77557`

Description

===========

Method :php:`QueryView->tableWrap()` has been marked as deprecated.

Impact

======

Extensions using this method will trigger a deprecation log entry.

Affected Installations

======================

Extensions using :php:`QueryView->tableWrap()`

Migration

=========

Use :php:`'<pre>' . $str . '</pre>'` instead.

.. index:: PHP-API, Backend

### PR DESCRIPTION
**Pull request by tomasnorre**
Resolves #578

This is still WIP, but pushed in the hope of getting some help :)

### CODE REVIEW

File: src/Rector/v8/v3/RefactorQueryViewTableWrapRector.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        if (! $this->isMethodStaticCallOrClassMethodObjectType($node, QueryView::class)) {
+            return null;
+        }
+        if (! $this->isName($node->name, 'tableWrap')) {
+            return null;
+        }
+
+        ///** @var Arg[] $args */
+        //$args = $node->args;
+        //$firstArgument = array_shift($args);
```
Comment (tomasnorre) [inline src/Rector/v8/v3/RefactorQueryViewTableWrapRector.php:42]: Again having hard time figuring out how to wrap the `$firstArgument` with `<pre>` similar to in https://github.com/sabbelasichon/typo3-rector/pull/1637

Comment (sabbelasichon) [reply src/Rector/v8/v3/RefactorQueryViewTableWrapRector.php:42]: What you need in this case it to return a node of type Concat
return new Concat(new Concat(new String_('<pre>'), $firstArgument->value), new String_('</pre>'));

Comment (sabbelasichon) [reply src/Rector/v8/v3/RefactorQueryViewTableWrapRector.php:42]: This look really ugly, but it is the way how it works. Getting nodes and writing nodes all the way down. You have to get used to it.

Comment (tomasnorre) [reply src/Rector/v8/v3/RefactorQueryViewTableWrapRector.php]: It's a little better without the full namespaces :)

**Pull request merged**